rules/mobile_inference_rules.adoc

```diff
@@ -153,7 +153,8 @@ described in the table below. The number of queries is selected to ensure suffic
 |MobileNetEdge - Single stream |ImageNet 2012 validation data set (50000 images) | http://image-net.org/challenges/LSVRC/2012/ | 98% of FP32 (76.19%)
 |MobileNetEdge - Offline |ImageNet 2012 validation data set (50000 images) | http://image-net.org/challenges/LSVRC/2012/ | 98% of FP32 (76.19%)
 |MobileDet-SSD - Single stream |MS-COCO 2017 validation set (5000 images) | http://images.cocodataset.org/zips/val2017.zip | 95% of FP32 (mAP 0.285)
-|DeepLab v3 - Single stream |ADE20K val set (2000 images) | http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip | 97% of FP32 (mIOU 54.8% 32 classes)
+|MOSAIC - Single stream |ADE20K val set (2000 images) | http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip | 96% of FP32 (mIOU 59.75% 32 classes)
```

Comment:
@willc2010 MOSAIC accuracy number is stated as 59.8% on the website [here](https://mlcommons.org/en/inference-mobile-20/) . Although its a small diff , I think we should keep it consistent across. Not sure which one is right though.

Reply:
Good eye, I fixed it